packages/cli/test/build.test.ts

```diff
@@ -1,36 +1,9 @@
 import { describe, expect, it } from 'vitest';
-import fs from 'node:fs';
 import { execa } from 'execa';
 
 const cmd = '../../../bin/cli.js';
 
 describe('tz build', () => {
-  it('default', async () => {
-    const cwd = new URL('./fixtures/build-default/', import.meta.url);
-    await execa('node', [cmd, 'build'], { cwd });
-    const testFile = new URL('./tokens/index.css', cwd);
```

Comment:
Turborepo was right for calling this out: we’re relying on a downstream package to run this test, which is backwards. Moved to plugin-css so it can be run more-easily